### ISSUE TITLE
Have you Created Test plan ever? If Yes then How you created??

### ISSUE THREAD

**Issue by Roshani26149** [VisionITTesting/question-and-answer-forum#21]

**Comment by akashdktyagi** [comment]
@Roshani26149 Frankly, I have never created a test plan i my 11 years of exp. Previously, some test managers had to create it, that to only in the beginning of the project. And in this agile way of working the importance of test plan are more or less diminished. In agile all you have is features and user stories and you have to test each and every aspect of the user story. That is what mid level testers and automation testers usually do. 
Having said that test plans and test strategies also play important role in documenting what are the things you will be testing and how you will be testing them, scopes, methodologies etc. But as the project moves ahead there is very little relevance to these documents. In agile you move fast and you adapt fast, you can not hold yourself with some old test plan document and strategies listed years ago, neither you would like to take pain keeping a old document updated.

So, bottom line is you should know what is a "Test Plan"; you should know what is a "Test Strategy" document. But thats it. You usually do not have to create it or worry about it in agile.
